examples/main.go

```diff
@@ -23,15 +23,269 @@ func main() {
 		log.Fatalf("did not connect: %v", err)
 	}
 	defer conn.Close()
-	c := pb.NewQdrantClient(conn)
+
+	// create grpc collection client
+	collections_client := pb.NewCollectionsClient(conn)
 
 	// Contact the server and print out its response.
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	defer cancel()
-	r, err := c.HealthCheck(ctx, &pb.HealthCheckRequest{})
+
+	// Check Qdrant version
+	qdrantClient := pb.NewQdrantClient(conn)
+	healthCheckResult, err := qdrantClient.HealthCheck(ctx, &pb.HealthCheckRequest{})
+	if err != nil {
+		log.Fatalf("Could not get health: %v", err)
+	} else {
+		log.Printf("Qdrant version: %s", healthCheckResult.GetVersion())
+	}
+
+	collectionName := "test_collection"
```

Comment:
maybe moving these var definitions into the var block --just next to `addr`.

Reply:
done